section-2/exercises/dKozlovskyi.js

```diff
@@ -0,0 +1,43 @@
+//Exercise 1:
+let currentTemperature;
```

Comment:
Use **let** when you know that this variable will be reassigned several times during code execution.  In this case currentTemperature, isRaining , cityName should be decelerated as **const** instead of **let**.